docs/guide.rst

```diff
@@ -1921,15 +1921,14 @@ Dependabot
 
 Dependabot_ creates pull requests with automated dependency updates.
 
-Follow these steps to set up Dependabot for your repository:
+Enable `Security Updates` from the `Security` tab on the repository under `Dependabot alerts` > `Dependabot Security Updates`.
```

Comment:
Is this still required to receive dependency updates?

According to the [documentation],

> You enable GitHub Dependabot version updates by checking a configuration file in to your repository.

[documentation]: https://docs.github.com/en/github/administering-a-repository/about-github-dependabot

So I think we can simply remove the line.

```suggestion
```